pkg/controllers/cloud/cloudentityselector_controller.go

```diff
@@ -47,14 +48,65 @@ type CloudEntitySelectorReconciler struct {
 	Scheme               *runtime.Scheme
 	selectorToAccountMap map[types.NamespacedName]types.NamespacedName
 	Poller               *Poller
+	NPController         NetworkPolicyController
+	initialized          bool
+	Cache                map[types.NamespacedName]struct{}
```

Comment:
shouldnt this be private variable?

Reply:
We need to allocate the Cache map in the main.go, where we init the CloudEntitySelectorReconciler.
Will move it to SetupWithManager function, so that it can be private to the package.